Automattic/wp-calypso
# Editor: Use native preview on Block Editor.

## Changes proposed in this Pull Request

This PR removes Calypso's preview button override and re-enables Gutenberg's native preview dropdown menu.

**When deploying, first deploy WBE, then deploy Calypso.**

## Testing instructions

* Run `yarn start`.
* Run `yarn dev --sync` on `apps\editing-toolkit`.
* Sandbox a site.
* Sandbox `widgets.wp.com`.
* Go to `calypso.localhost:3000/post/your_sandboxed_site.wordpress.com`.
* Click **Preview** button.
* It should use native preview. It should not show a preview modal.
* E2E tests should not fail in this PR.

## Screenshot

**Before:**

https://user-images.githubusercontent.com/1287077/136546078-cb40a62b-d54c-4367-bc1f-43f1cd6f60e4.mp4

**After:**

https://user-images.githubusercontent.com/1287077/136546089-7547e8f6-fea7-46af-8164-8201db419561.mp4

Fixes https://github.com/Automattic/wp-calypso/issues/56834
